src/couch/src/couch_debug.erl

```diff
@@ -326,6 +328,17 @@ help(print_table) ->
           - TableSpec: List of either {Value} or {Width, Align, Value}
             where Align is either left/center/right.
 
+        ---
+    ", []);
+help(print_report) ->
+    io:format("
+        print_report(Report)
+        --------------------------------
+
+        Print a report in table form.
+          - Report: List of {InfoKey, [{InfoKey, InfoVal}]} where each InfoKey is unique
```

Comment:
Why do we need to specify `InfoKey` twice? Can we construct it in `print_report`?

```
    lists:map(
        fun({InfoKey, Value}) ->
            TableSpec1 = [
                {50, centre, info},
                {100, right, InfoKey}
            ],
            io:format("~s~n", [table_row(InfoKey, 2, {InfoKey, Value}, TableSpec1)])
        end,
        Report
    ).
```

Reply:
The `table_row` function expects the data to be structured that way from what I could see.

Reply:
there is a line change in a snippet

```
- io:format("~s~n", [table_row(InfoKey, 2, Value, TableSpec1)])
+ io:format("~s~n", [table_row(InfoKey, 2, {InfoKey, Value}, TableSpec1)])
```

Reply:
It is possible to build it in `print_report` instead. I will move it and simplify the `Report` data structure.